### DISCUSSION TITLE
Recording stopped, no retry logic implemented yet?

### DISCUSSION THREAD

**Discussion by 0ip** [bluenviron/mediamtx#2438] [Questions] [answered]
I had just noticed that two of my four cameras had stopped recording. The folders belonging to them were deleted and apparently no attempts were made to restart the recording. Is it up to the user to deal with such situations, or is this not yet implemented?

Log showed:
- ERR [path 64/high] [record] DTS is not monotonically increasing, was 21h32m36.01s, now is 21h32m36.01s
- WAR [path 63/high] [RTSP source] 3 RTP packets lost

**Comment by aler9** [comment, marked as answer]
moved in an issue here: #2439
